internal/distro/distro.go

```diff
@@ -121,7 +121,7 @@ type ImageType interface {
 	// specified in the given blueprint; it also returns any warnings (e.g.
 	// deprecation notices) generated by the manifest.
 	// The packageSpecSets must be labelled in the same way as the originating PackageSets.
-	Manifest(b *blueprint.Customizations, options ImageOptions, repos []rpmmd.RepoConfig, packageSpecSets map[string][]rpmmd.PackageSpec, containers []container.Spec, seed int64) (Manifest, []string, error)
+	Manifest(b *blueprint.Customizations, options ImageOptions, repos []rpmmd.RepoConfig, packageSpecSets map[string][]rpmmd.PackageSpec, containers []container.Spec, customRepos []rpmmd.RepoConfig, seed int64) (Manifest, []string, error)
```

Comment:
I think this was because I was trying to do some cleverness with the file resolver and passing the keys down, but I don't think there'll be any need for this with the simplifications... so I'll revert back to using the customizations.